repo/packages/M/marathon/9/config.json

```diff
@@ -0,0 +1,378 @@
+          "type" : "string"
+        },
+        "plugin-dir" : {
+          "description" : "Path to a local directory containing plugin jars",
+          "type" : "string"
+        },
+        "reconciliation-initial-delay" : {
+          "description" : "This is the length of time, in milliseconds, before Marathon begins to periodically perform task reconciliation operations",
+          "type" : "integer",
+          "default" : 15000
+        },
+        "reconciliation-interval" : {
+          "description" : "This is the length of time, in milliseconds, between task reconciliation operations.",
+          "type" : "integer",
+          "default" : 300000
+        },
+        "reporter-datadog" : {
+          "description" : "URL to dogstatsd agent. e.g. udp://localhost:8125?prefix=marathon-test&tags=marathon&interval=10",
+          "type" : "string"
+        },
+        "reporter-graphite" : {
+          "description" : "URL to graphite agent. e.g. tcp://localhost:2003?prefix=marathon-test&interval=10",
+          "type" : "string"
+        },
+        "revive-offers-repetitions" : {
+          "description" : "Repeat every reviveOffer request this many times, delayed by the --min_revive_offers_interval.",
+          "type" : "integer",
+          "default" : 3
+        },
+        "save-tasks-to-launch-timeout" : {
+          "description" : "Timeout (ms) after matching an offer for saving all matched tasks that we are about to launch. When reaching the timeout, only the tasks that we could save within the timeout are also launched. All other task launches are temporarily rejected and retried later.",
+          "type" : "integer",
+          "default" : 3000
+        },
+        "scale-apps-initial-delay" : {
+          "description" : "This is the length of time, in milliseconds, before Marathon begins to periodically attempt to scale apps",
+          "type" : "integer",
+          "default" : 15000
+        },
+        "scale-apps-interval" : {
+          "description" : "This is the length of time, in milliseconds, between task scale apps.",
+          "type" : "integer",
+          "default" : 300000
+        },
+        "ssl-keystore-password" : {
+          "description" : "Password for the keystore supplied with the `ssl_keystore_path` option. Required if `ssl_keystore_path` is supplied. May also be specified with the `MESOSPHERE_KEYSTORE_PASS` environment variable.",
+          "type" : "string"
+        },
+        "ssl-keystore-path" : {
+          "description" : "Path to the SSL keystore. HTTPS (SSL) will be enabled if this option is supplied. Requires `--ssl_keystore_password`. May also be specified with the `MESOSPHERE_KEYSTORE_PATH` environment variable.",
+          "type" : "string"
+        },
+        "store-cache" : {
+          "description" : "Enabled: Enable an in memory cache for the storage layer. Disabled: Disable the in memory cache for the storage layer ",
```

Comment:
This description doesn't match the one of the other boolean flags. It doesn't say what's the default.